tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -207,6 +207,9 @@
           ansiblePort: 22
           ansibleUser: root
           ansibleVars:
+            os_net_config_iface: {{ dataplane_os_net_config_iface | default ('nic1') }}
+            os_net_config_routes: {{ dataplane_os_net_config_routes | default(ctlplane_host_routes) }}
```

Comment:
`ctlplane_host_routes` is created on dataplane-operator reconcile loop

Reply:
https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/pkg/deployment/inventory.go#L114

Reply:
so you mean, i should not try to default that here it doesn't exist yet. perhaps |default(omit) will work instead here then.

and I can have |default ctlplane_host_routes below in the template, as before WDYT?

Reply:
exactly!

Reply:
and same for dnsnameservers too then i think https://github.com/openstack-k8s-operators/dataplane-operator/blob/0874b6a673e9fcc6ac7e96896bf3a7047b05fe45/pkg/deployment/inventory.go#L113

Reply:
thanks for reviewing @fao89 ++